runtime/test/manifest-test.js

```diff
@@ -1283,6 +1283,41 @@ resource SomeName
     assert(recipe.normalize());
     assert(recipe.isResolved());
   });
+  it('can resolve a particle with a schema reference', async () => {
+    let manifest = await Manifest.parse(`
+      schema Foo
+        Text far
+      particle P
+        in Bar {Reference<Foo> foo} bar
+      recipe
+        create as handle
+        P
+          bar = handle
+    `);
+
+    let [recipe] = manifest.recipes;
+    assert.equal(manifest.particles[0].toString(),
+`particle P in 'null'
+  in Bar {Reference<Foo {Text far}> foo} bar
+  affordance dom`);
```

Comment:
This looks brittle... can you use the same asserts on normalize() and isResolved() as the other tests, and/or inspect the manifest object to check that the handles is correctly a reference?

Reply:
Not yet, because I haven't built out what a reference is completely within schemas. Maybe a TODO to switch to that when we can?

Reply:
SGTM